metron-platform/metron-parsers/src/test/java/org/apache/metron/parsers/csv/CSVParserTest.java

```diff
@@ -87,15 +87,15 @@ public void test() throws IOException {
       Assert.assertEquals("grok", o.get("col3"));
     }
     {
-      String line = "foo, bar, grok";
+      String line = " foo , bar , grok ";
```

Comment:
This should really be implemented as a separate test function (additional) since we're adding functionality here. you can definitely use the same form, but we're testing a different unit of functionality.